easybuild/scripts/bootstrap_eb.py

```diff
@@ -519,6 +532,8 @@ def main():
 # EasyBuild is a (set of) Python packages, so it depends on Python
 # usually, we want to use the system Python, so no actual Python dependency is listed
 allow_system_deps = [('Python', SYS_PYTHON_VERSION)]
+
+preinstallopts = '%(preinstallopts)s'
```

Comment:
not sure what you mean?

these?

v1.16.2: https://github.com/hpcugent/easybuild-easyconfigs/pull/1457
v2.0.0: https://github.com/hpcugent/easybuild-easyconfigs/pull/1456